src/popup/router/pages/FungibleTokens/TokenDetails.vue

```diff
@@ -52,44 +53,134 @@
       class="token-info"
     >
       <DetailsRow
-        :label="$t('pages.token-details.abbreviation')"
+        :label="$t('pages.token-details.token')"
         :text="tokenData.symbol"
       />
       <DetailsRow
+        v-if="tokenData.decimals"
         :label="$t('pages.token-details.decimals')"
         :text="tokenData.decimals"
       />
       <DetailsRow
-        v-if="tokenData.contractId"
-        :class="{ contract: tokenData.contractId }"
+        v-if="tokenData.contract"
         :label="$t('pages.token-details.contract')"
-        :text="tokenData.contractId"
-      />
+      >
+        <template #text>
+          <AddressShortening :address="tokenData.contract" />
```

Comment:
This is suppose to be a contractId after you have rebased on top of https://github.com/aeternity/superhero-wallet/pull/1494, carefully check other places.